packages/storybook/stories/INTERNAL/Chat/InputBox.stories.tsx

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { ITextField, Stack, mergeStyles } from '@fluentui/react';
+import { Mention, FluentThemeProvider, MessageThread, useTheme } from '@internal/react-components';
+import { Title, Description, Props, Heading, Canvas } from '@storybook/addon-docs';
+import { Meta } from '@storybook/react/types-6-0';
+import React, { useState } from 'react';
+import { InputBoxComponent } from '../../../../react-components/src/components/InputBoxComponent';
+import { borderAndBoxShadowStyle } from '../../../../react-components/src/components/styles/SendBox.styles';
+import { COMPONENT_FOLDER_PREFIX } from '../../constants';
+import { controlsToAdd, hiddenControl } from '../../controlsUtils';
+import { GetHistoryChatMessages } from '../../MessageThread/snippets/placeholdermessages';
+
+const getDocs: () => JSX.Element = () => {
+  return (
+    <>
+      <Title>InputBox</Title>
+      <Description>InputBox is a component that allows users to type text and mentions.</Description>
+      <Heading>Example</Heading>
+      <Canvas>
+        <InputBoxStory />
+      </Canvas>
+
+      <Heading>Props</Heading>
+      <Props of={InputBoxComponent} />
+    </>
+  );
+};
+
+const InputBoxStory = (): JSX.Element => {
+  const exampleHtmlMessage =
+    "<p>Hi<p>Paragraph</p><msft-mention id='2'>Patricia Adams</msft-mention> <span>and <it><em>and</em></it> </span> and you <msft-mention id='3'>Person 1</msft-mention>!</p><p>Test <self-closing/></p>";
+  const sendTextFieldRef = React.useRef<ITextField>(null);
+  const theme = useTheme();
+  const [textValue, setTextValue] = useState<string>(exampleHtmlMessage);
+  const trigger = '@';
+  const suggestions: Mention[] = [
+    {
+      id: '1',
+      displayText: ''
+    },
+    {
+      id: '2',
+      displayText: 'Patricia Adams'
+    },
+    {
+      id: '3',
+      displayText: '1'
+    },
+    {
+      id: '4',
+      displayText: 'Your user'
+    }
+  ];
+  return (
+    <FluentThemeProvider>
+      <MessageThread userId={'4'} messages={GetHistoryChatMessages()} />
+      <div style={{ width: '31.25rem', height: '20rem' }}>
+        <Stack
+          className={mergeStyles(
+            borderAndBoxShadowStyle({
+              theme,
+              hasErrorMessage: false,
+              disabled: false
+            })
+          )}
+        >
+          <InputBoxComponent
```

Comment:
I'm seeing an error here where the property `children` is not being assigned

Reply:
Interesting that CI does not see it.